config/locales/ja.yml

```diff
@@ -273,7 +273,7 @@ ja:
       content: セキュリティ認証に失敗しました。Cookieをブロックしていませんか？
       title: セキュリティ認証に失敗
     '429': リクエストの制限に達しました。
-    noscript: Mastodonのウェブアプリケーションを利用する場合はJavaScriptを有効にしてください。またはあなたのプラットフォーム向けのMastodonネイティブアプリを探すことができます。
+    noscript_html: Mastodonのウェブアプリケーションを利用する場合はJavaScriptを有効にしてください。またはあなたのプラットフォーム向<a href="https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/Apps.md">けのMastodonネイティブアプリ</a>を探すことができます。
```

Comment:
broken Japanese translate.

Do not add the A tag except for the locale you know (the translator for each language should do it).